### ISSUE TITLE
release 0.1.0

### ISSUE THREAD

**Issue by WSH032** [WSH032/pytauri#4]
Blocked by:

- [x] Documentation
    - [x] python api docs: #7
    - [x] rust api docs: #6 
    - [x] Usage docs: #28 
- [x] CI
    - [x] Test: #8 
    - [x] Release
        - [x] rust: #9 
        - [x] python: #19 
        - [x] js: #25 

Pending issues:

- [ ] Version strategy for Rust and its Python bindings
- [ ] Version strategy for pytauri core and plugins

Can be postponed:

- [ ] CONTRIBUTING.md

**Comment by WSH032** [comment]
Release [v0.1.0-beta](https://github.com/WSH032/pytauri/releases/tag/v0.1.0-beta).

Pending and postponed issues will be resolved in `v0.2`.
